datalad/utils.py

```diff
@@ -1193,13 +1021,6 @@ def wrapped_func(*args, **kwargs):
 #
 
 
-# unused in -core
-@contextmanager
-def nothing_cm():
```

Comment:
oh -- here you are my little helper I am going to use in the #749 to avoid code duplication for either doing chdir or not!
I guess I could reintroduce it there if keeping it here is a big ask